### PR TITLE
Doc: add readme for ccnp sdk

### PR DESCRIPTION
**Pull request by Ruoyu-y**
* Add the overall readme for ccnp sdk

### CODE REVIEW

**Comment by Ruoyu-y** [comment]
Adding the architecture for readme. @dongx1x @hairongchen @hjh189 can add more in the 'Quote', 'End-to-end examples' and 'Troubleshooting'.

File: sdk/python3/README.md [hunk 1]
```diff
@@ -1,19 +1,174 @@
+# Confidential Cloud Native Primitives client library for Python
```
Comment (hwang37) [inline sdk/python3/README.md:1]: Confidential Cloud-Native Primitives client library for Python

BTW: a generation comment: Cloud-Native is used as in the name of CCNP, so please search/replace it in CCNP context.

File: sdk/python3/README.md [hunk 2]
```diff
@@ -1,19 +1,174 @@
...
+[Source code][source_code]
+| [Package (PyPI)][ccnp_pypi]
+| [API reference documentation][api_doc]
 
-Here provides the sdks for confidential cloud native primitives which will ease the usage
-of confidential computing using TEE.
+## Getting started
 
+### Prerequisites
+In order to work properly, user need to have the backend services ready on the TEE or TPM enabled platform. Please refer to each deployment guide reside in the [service](../../service/) folder to install the backend services.
```
Comment (hwang37) [inline sdk/python3/README.md:16]: ...or TPM enabled platform first.

File: sdk/python3/README.md [hunk 3]
```diff
@@ -1,19 +1,174 @@
...
+Using this sdk, user could fetch various measurements from different perspective and categories.
+Basic support on measurement focus on the platform measurements, including TEE report, values within TDX RTMR registers or values reside in TPM PCR registers.
+There's also advanced support to provide measurement for a certain workload or container. The feature is still developing in progress.
+
+#### MeasurementType for platform
+
+The measurement sdk supports fetching different types of evidence depending on the environment.
+Currently, CCNP supports the following categories of measurements:
+
+* TYPE_TEE_REPORT - This provides the report fetching on various Trusted Execution Environment from all kinds of vendors, including Intel TDX, AMD SEV (WIP), etc.
```
Comment (hwang37) [inline sdk/python3/README.md:51]: WIP => Working In Progress

(To clarify that it's not a technical term)

**Review by hwang37 — COMMENTED** [review]
A general comment:
Cloud-Native (with '-') is used as in the official name of CCNP, so please search/replace it in readme during CCNP context.

**Review by hairongchen — APPROVED** [review]
LGTM

**Pull request merged**